sdcm/nemesis/monkey/manager.py

```
@@ -0,0 +1,419 @@
+                location,
+            ]
+        )
+
+    assert mgr_task is not None, "Backup task wasn't created"
+
+    status = mgr_task.wait_and_get_final_status(timeout=54000, step=5, only_final=True)
+    runner.actions_log.info(f"Scylla Manager backup task finished with status: {status}")
+    if status == TaskStatus.DONE:
+        runner.log.info("Task: %s is done.", mgr_task.id)
+    elif status in (TaskStatus.ERROR, TaskStatus.ERROR_FINAL):
+        assert False, f"Backup task {mgr_task.id} failed"
```

Ruff (0.15.21)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)